website/docs/addon_applications_admin.md

```diff
@@ -1,26 +1,235 @@
+2. **Tool Variants:**
+   1. **Name:** Tool variant name.
+   2. **Label:** Tool variant label. This name will also be used as the tool variant setting name.
+   3. **Hosts:** create a list of host names you want the tool variant to work with.
+   4. **Applications:** select which applications you want the tool variant to work with.
+   5. **Environments:** tool variant environments
+   6. **+ button:** To add more tool variants.
+3. **+ button:** To add more tool groups.
+
+:::tip Tool Examples
+
+AYON comes with some major CG renderers pre-configured as an example, but these and any others will need to be changed to match your particular environment.
+
+![](assets/applications/examples_tools.png)
+
+<details><summary>Pre-configured Example - Autodesk Arnold</summary>
+
+![tools](assets/applications/arnold_tool_example.png)
+
+In this example you can see that we set most of the environment variables in the general MTOA level, and only specify the version variable in the individual versions below. Because all environments within AYON setting will resolve any cross references, this is enough to get a fully dynamic plugin loading as far as your folder structure where you store the plugins is nicely organized.
+
+MTOA will automatically set the `MAYA_VERSION`(which is set by Maya Application environment) and `MTOA_VERSION` into the `MTOA` variable. We then use the `MTOA` to set all the other variables needed for it to function within Maya.
+</details>
+:::
+
+### Tools filters
+Tools that will be used on application launch. It is profile based filtering that allows to filter tools for a context. Tools can have more specific filtering than applications.
+
+All the tools defined in Tools Definitions can then be assigned to projects. You can also change the tools versions on any project level all the way down to individual asset or shot overrides. So it is possible to upgrade you render plugin for a single shot, while not risking the incompatibilities on the rest of the project.
+
+
+![](assets/applications/tools_filters_profiles.png)
+
+1. **Folder Paths:** Create a list of folder paths to determine which ones the filter profile will affect. *It supports regex patterns.*
+2. **Task Types:** Choose from a list of task types to determine which ones the filter profile will affect.
+3. **Task Names:** Create a list of task names to determine which ones the filter profile will affect.
+4. **Tools:** Tools list to determine which ones the filter profile will affect.
+5. **+ button:** To add more applications filters profiles.
+
+:::tip Example Filter profile
+Find example of Tool defintion and a Filter profile in the FAQ section below:
+[How to make a tool available for all folder paths in a project for a specific application variant ?](addon_applications_admin#how-to-make-a-tool-available-for-all-folder-paths-in-a-project-for-a-specific-application-variant-)
+:::
+
+## FAQ
+### How to use application variant group settings ?
```

Comment:
What is "variant group" for you? Group is "Maya", variant is "2024", "variant group" is not a thing.

Reply:
just added by accident. it should be fixed now.